### PR TITLE
new Leveragelp Query

### PR DESCRIPTION
**Pull request by politeWall**
- [x] Leveragelp Open and close estimation 
- [x] MasterChef Params
- [x] MasterChef  Pool Info

### CODE REVIEW

File: scripts/queries.sh [hunk 1]
```diff
@@ -560,6 +586,15 @@ function get_estaking_rewards() {
...
 
+function master_chef_params() {
+    printf "\n# Get Masterchef Params\n"
+    query_contract \
+        "$ah_contract_address" \
+        '{
+        "get_masterchef_params": {}
+    }'
+}
+
```
Comment (amityadav0) [inline scripts/queries.sh:597]: can we add function for `master_chef_pool_info` ?

File: scripts/queries.sh [hunk 2]
```diff
@@ -532,6 +532,32 @@ function leveragelp_position() {
...
+            "collateral_amount": "1000000",
+            "amm_pool_id": 2,
+            "leverage": "5.0",
+        }
+    }'
+}
+
+# leveragelp close estimation
+function leveragelp_close_estimation() {
+    printf "\n# everagelp close estimation\n"
```
Comment (amityadav0) [inline scripts/queries.sh:550]:
```suggestion
    printf "\n# leveragelp close estimation\n"
```

File: scripts/queries.sh [hunk 3]
```diff
@@ -780,6 +821,9 @@ case "$2" in
 "get_estaking_rewards")
     get_estaking_rewards
     ;;
+"master_chef_params")
+    master_chef_params
+    ;;
```
Comment (amityadav0) [inline scripts/queries.sh:826]: do we need to add `master_chef_pool_info ` here too ?

File: bindings/src/query_resp.rs [hunk 4]
```diff
@@ -694,6 +694,67 @@ pub struct LeveragelpPoolsResponse {
...
+
+#[cw_serde]
+pub struct LeveragelpOpenEstResponse {
+    position_size: Int128,
+    weight_balance_ratio: Decimal,
+    borrow_fee: Decimal,
+}
+
+#[cw_serde]
+pub struct LeveragelpCloseEstResponse {
```
Comment (Wgil) [inline bindings/src/query_resp.rs:706]: This is wrong. See https://github.com/elys-network/elys/pull/544/files#diff-608a4d4096e8cad7b23065a30151bc5cb22c87e505600789c15cd71c3a0c8d11R229

**Review by Wgil — CHANGES_REQUESTED** [review]
Please check and fix both open and close estimation interfaces. They dont match

**Review by Wgil — APPROVED** [review]

**Pull request merged**